src/smart-components/role/add-role-new/review.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import useFormApi from '@data-driven-forms/react-form-renderer/dist/cjs/use-form-api';
+import {
+    Stack,
+    StackItem,
+    Grid,
+    GridItem,
+    Text,
+    TextVariants
+} from '@patternfly/react-core';
+import './review.scss';
+
+const mockData = [
+    { cells: [ 'cost:aws.account:read', 'Project 1' ]},
+    { cells: [ 'cost:azure:read', 'Cluster 1, Cluster 2, Cluster 3' ]},
+    { cells: [ 'cost:aws.account:write', 'Project 1' ]},
+    { cells: [ 'cost:aws.account:execute', 'Project 1' ]},
+    { cells: [ 'cost:aws:something', 'Project 1' ]},
+    { cells: [ 'cost:aws:something2', 'Project 2' ]},
+    { cells: [ 'cost:aws:something3', 'Project 3' ]},
+    { cells: [ 'cost:aws:something4', 'Project 4' ]},
+    { cells: [ 'cost:aws:something5', 'Project 5' ]},
+    { cells: [ 'cost:aws:something6', 'Project 6' ]},
+    { cells: [ 'cost:aws:something7', 'Project 7' ]},
+    { cells: [ 'cost:aws:something8', 'Project 8' ]},
+    { cells: [ 'cost:aws:something9', 'Project 9' ]},
+    { cells: [ 'cost:aws:something10', 'Project 10' ]},
+    { cells: [ 'cost:aws:something11', 'Project 11' ]},
+    { cells: [ 'cost:aws:something12', 'Project 12' ]}
+];
+
+const stickyTable = (columns, rows) => (
+    <div className='ins-c-rbac__sticky'>
+        <Grid className="title">
```

Comment:
Can you name these classes a bit more direct? `ins-c-rbac__sticky--title` and `ins-c-rbac__sticky--data`